.azure-pipelines/templates/package-binaries.yml

```
@@ -3,11 +3,19 @@ parameters:
     default: linux
   - name: zipcommand
     default: "zip -9rv"
+  - name: useMinGW
+    default: false
 
 steps:
   - bash: |
       set -ex
       VERSION=$(cat version)
+      if [[ "${{ parameters.useMinGW }}" == "true" ]]; then
+        MINGWLIB="C:\ProgramData\chocolatey\lib\mingw\tools\install\mingw64\bin\\"
+        cp $MINGWLIB\libquadmath-0.dll bin/
+        cp $MINGWLIB\libwinpthread-1.dll bin/
+        cp $MINGWLIB\libgcc_s_seh-1.dll bin/
+      fi
```

Might be nicer to break this up into two steps as follows:
```suggestion
  - bash: |
      set -ex
      MINGWLIB="C:\ProgramData\chocolatey\lib\mingw\tools\install\mingw64\bin\"
      cp $MINGWLIB\libquadmath-0.dll bin/
      cp $MINGWLIB\libwinpthread-1.dll bin/
      cp $MINGWLIB\libgcc_s_seh-1.dll bin/
    condition: eq('${{ parameters.useMinGW }}', true)
    displayName: 'Package MinGW Libs'
  - bash: |
      set -ex
      VERSION=$(cat version)
```